firefox-ios/Client/Frontend/PasswordGenerator/PasswordGeneratorTelemetry.swift

```diff
@@ -0,0 +1,16 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Glean
+
+public struct PasswordGeneratorTelemetry {
+    func passwordGeneratorDialogShown() {
+        GleanMetrics.PasswordGenerator.shown.add()
+    }
+
+    func usePasswordButtonPressed() {
+        GleanMetrics.PasswordGenerator.filled.add()
+    }
+}
```

Comment:
Where do you plan to use this?

Reply:
Not too sure, just waiting on product to get back to me to figure out how the data would be used. Once I hear back from them I'll fill out the data review form and tag Roux :)